src/qtmozembed/declarativewebpage.cpp

```diff
@@ -1,6 +1,8 @@
 /****************************************************************************
 **
-** Copyright (C) 2014 Jolla Ltd.
+** Copyright (C) 2014-2019 Jolla Ltd.
```

Comment:
Lower case "(c)" also I prefer spaces around the - -mark

Reply:
Changed.